src/command.rs

```diff
@@ -475,6 +475,7 @@ pub fn parse(frame: Frame) -> Result<Command, ParseCommandError> {
         "ttl" => Ok(Command::Ttl(Ttl::parse(&mut parser)?)),
         "ping" => Ok(Command::Ping),
         "keyspaces" => Ok(Command::Keyspaces),
+        "export" => Ok(Command::Keyspaces),
```

Comment:
I don't think we need to add a new command, the keyspaces command should be used to construct the export. Basically have a flag on the frontend (CLI) to read the data sent from the `keyspaces` command and constuct the export. maybe something like this 
```shell
keyspaces --export
```